addons/account/models/account_rounding.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+from odoo.tools import float_round
+
+
+class AccountRounding(models.Model):
+    """
+    In some countries, we need to be able to make appear on an invoice a rounding line, appearing there only because the
+    smallest coinage has been removed from the circulation. For example, in Switerzland invoices have to be rounded to
+    0.05 CHF because coins of 0.01 CHF and 0.02 CHF aren't used anymore.
+    see https://en.wikipedia.org/wiki/Cash_rounding for more details.
+    """
+    _name = 'account.rounding'
+    _description = 'Account Rounding'
+
+    name = fields.Char(string='Name',translate=True, required=True)
+    rounding = fields.Float(string='Rounding Precision', required=True,
+        help='Represent the non-zero value smallest coinage (for example, 0.05).')
+    account_id = fields.Many2one('account.account', string='Account', required=True)
```

Comment:
account_id is not useful if we want to modify the biggest tax, I suppose... so it shouldn't be required.